Pomodoro.Api/ViewModels/Statistics/AnalyticsPerMonthViewModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿// <copyright file="AnalyticsPerMonthViewModel.cs" company="PomodoroGroup_GL_BaseCamp">
+// Copyright (c) PomodoroGroup_GL_BaseCamp. All rights reserved.
+// </copyright>
+
+namespace Pomodoro.Api.ViewModels.Statistics
+{
+    /// <summary>
+    /// Represents a view model which includes analytics data per one month.
+    /// Used for the <see cref="AnnualStatisticsViewModel"/>.
+    /// </summary>
+    public class AnalyticsPerMonthViewModel
+    {
+        /// <summary>
+        /// Gets or sets a month number (1-12) for which analytics is gathered.
```

Comment:
I think it would be useful to create an enum for months and use it instead of int